tests/extensions/TestHook.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace Appwrite\Tests;
+
+use PHPUnit\Runner\AfterTestHook;
+
+class TestHook implements AfterTestHook
+{
+    public function executeAfterTest(string $test, float $time): void
+    {
+        printf("\n%s ended in %s seconds", 
```

Comment:
`\n%s ended in %s seconds` => `%s ended in %s seconds\n` = should look a bit better in the terminal logs

Reply:
locally, that looked weird, I'll update